.gitpod.Dockerfile

```diff
@@ -1,7 +1,8 @@
-FROM gitpod/workspace-base
+FROM gitpod/workspace-full-vnc
 
 ARG PYTHON_VERSION=3.9
 
 RUN sudo add-apt-repository ppa:deadsnakes/ppa -y
-RUN sudo apt install python${PYTHON_VERSION} -y 
-RUN sudo apt install python${PYTHON_VERSION}-venv -y
\ No newline at end of file
+RUN sudo apt install python${PYTHON_VERSION} -y
+RUN sudo apt install python${PYTHON_VERSION}-venv -y
+RUN sudo apt install python${PYTHON_VERSION}-tk -y
```

Comment:
Does `setup.py` need to be updated to include any extra dependencies?

Reply:
Nope, Tkinter is apparently a python sys package and not a pypi one.